tools/plugins/com.liferay.ide.sdk.core/src/com/liferay/ide/sdk/core/SDK.java

```diff
@@ -63,6 +63,8 @@ public static String createXMLNameValuePair( String name, String value )
 
     public static List<String> SUPPORT_SERVER_TYPES = Arrays.asList( new String[]{ "tomcat", "jboss", "glassfish", "jetty"} );
```

Comment:
this should be found out dynamically not statically.  Because if we add a new server type in the future we have to modify this line.